owner-onboarding-server/src/main.rs

```diff
@@ -138,10 +157,51 @@ async fn report_to_rendezvous(udt: OwnerServiceUDT) -> Result<()> {
     Ok(())
 }
 
+async fn check_registration_window(udt: &OwnerServiceUDT) -> Result<()> {
+    let now_plus_window =
+        time::OffsetDateTime::now_utc().unix_timestamp() + (udt.ov_re_registration_window as i64);
+    // these are the ovs whose registration time will end and we need to
+    // re-register them
+    let ovs = udt
+        .ownership_voucher_store
+        .query_ovs_db_to2_performed_to0_less_than(false, now_plus_window)
+        .await?;
+    for ov in ovs {
+        match report_ov_to_rendezvous(
+            &ov,
+            &udt.owner_addresses,
+            &udt.owner_key,
+            udt.ov_registration_period,
+        )
+        .await
+        {
+            Ok(wait_seconds) => {
+                udt.ownership_voucher_store
+                    .store_metadata(
+                        ov.header().guid(),
+                        &fdo_store::MetadataKey::Local(
+                            OwnershipVoucherStoreMetadataKey::To0AcceptOwnerWaitSeconds,
+                        ),
+                        &time::Duration::new(wait_seconds.into(), 0),
```

Comment:
The rendezvous server may return a `wait_seconds` which is less than `ov_registration_period`. Maybe we should also check that is also bigger than `ov_re_registration_window` and  print a warning in the log saying that the re registration will not work for that ownership voucher and/or maybe disable `window_check_enabled`?

Reply:
Good point will add the warning. The `wait_seconds` that comes from the redezvous should be what the owner is requesting, _or less_, the only config that the rendezvous has is an upper limit, that we cannot know (this is just in our implementation).